### PR TITLE
A handful of fixes

### PR DESCRIPTION
**Pull request by raskchanky**
This fixes a handful of issues I noticed and/or encountered while trying to deploy our prod web site as part of the 0.81.0 release.  Please see the commit messages for full details.

### CODE REVIEW

File: www/Makefile [hunk 1]
```diff
@@ -47,24 +47,15 @@ template_reference: node_modules/json-schema-ref-parser
...
-ifndef AWS_SECRET_ACCESS_KEY
-	$(error AWS_SECRET_ACCESS_KEY is undefined)
-endif
 ifndef FASTLY_API_KEY
 	$(error FASTLY_API_KEY is undefined)
 endif
 ifndef FASTLY_SERVICE_KEY
 	$(error FASTLY_SERVICE_KEY is undefined)
 endif
-ifndef NETLIFYKEY
```
Comment (eeyun) [inline www/Makefile]: Netlify before was explicitly for deploying the site to acceptance. I think we disabled sentinels but have we sound an acceptance solution to replace this?

Comment (raskchanky) [reply www/Makefile]: IIRC, it's on @scotthain's roadmap, but right now, deploying from your local machine is the only option.  This change doesn't break that though.  It just means that if all you want to do is deploy the site to prod, you don't need to figure out your netflify key.

File: www/README.md [hunk 2]
```diff
@@ -98,6 +43,19 @@ Static site content for www.habitat.sh
   > Note: values for each of these can be found by logging into your AWS and Fastly control panels.
   > Note: make sure that the AWS credentials you use are for the Habitat AWS account and not the regular Chef AWS account.
 
+### A slight winkle
```
Comment (baumanj) [inline www/README.md]: [winkle](https://en.wikipedia.org/wiki/Winkle)?
image

Comment (raskchanky) [reply www/README.md]: :joy:

Comment (scotthain) [reply www/README.md]: fun fact - my wife's grad school roommate had multiple pet snails. large ones.

**Pull request merged**